ci/eval/compare/default.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  jq,
+  runCommand,
+  writeText,
+  supportedSystems,
+  ...
+}:
+{ beforeResultDir, afterResultDir }:
+let
+  inherit (import ./utils.nix { inherit lib; })
+    diff
+    groupByKernel
+    extractPackageNames
+    getLabels
+    ;
+
+  getAttrs = dir: builtins.fromJSON (builtins.readFile "${dir}/outpaths.json");
+  beforeAttrs = getAttrs beforeResultDir;
+  afterAttrs = getAttrs afterResultDir;
+
+  diffAttrs = diff beforeAttrs afterAttrs;
+
+  changed-paths =
+    let
+      rebuilds = lib.unique (diffAttrs.added ++ diffAttrs.changed);
+
+      rebuildsByKernel = groupByKernel rebuilds;
+      rebuildCountByKernel = lib.mapAttrs (
+        kernel: kernelRebuilds: lib.length kernelRebuilds
+      ) rebuildsByKernel;
```

Comment:
We could put those in the `toJSON` call below but it would require a `rec`. I don't know what is the recommended way.